src-docs/src/views/guidelines/accessibility.js

```diff
@@ -510,6 +510,56 @@ export default {
         </>
       ),
     },
+    {
+      title: 'Disabled elements',
+      wrapText: false,
+      text: (
+        <>
+          <EuiText grow={false}>
+            <p>
+              If you disable elements by adding the HTML{' '}
+              <EuiCode language="html" size="s">
```

Comment:
I don't think you need `size` here
```suggestion
              <EuiCode language="html">
```